schema/resolvers.js

```diff
@@ -92,7 +102,9 @@ const Resolvers = {
       ctx.repositories.QuestionPage.undelete(args.input.id),
 
     createAnswer: async (root, args, ctx) => {
-      const answer = await ctx.repositories.Answer.insert(args.input);
+      const defaultProperties = getDefaultAnswerProperties(args.input.type);
+      const input = merge({}, args.input, { properties: defaultProperties });
```

Comment:
I think we can get away with not setting default properties on Answer when it's created. When resolving Answer a field resolver for `properties` could do the merge at that point of the default properties and any overridden property values stored on the Answers table.

Reply:
On the other hand, setting properties on creation doesn't lead to unexpected consequences when updating defaults, plus the properties are going to be queried more often than they are created so perhaps makes more sense to leave as is.